app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -165,12 +172,13 @@ public void doPetSearch(SearchFilter searchFilter) {
     }
     public void onPetSearchSuccess(List<PetModel> pets) {
         mCardAdapter.addAll(pets);
- //       loadingProgress.setVisibility(View.GONE);
-        Log.d("FindActivity", "SUCCESS");
+        loadingProgressLayout.setVisibility(View.GONE);
+        Log.d("FindActivity", "SUCCESS Finding Pets");
     }
     public void onPetSearchError(String result) {
- //       loadingProgress.setVisibility(View.GONE);
-        Log.d("FindActivity", "ERROR");
+        Log.d("FindActivity", "ERROR FINDING PETS");
+        loadingMessageView.setText("Error finding pets, make sure you have an internet connection");
+        progressBar.setVisibility(View.INVISIBLE);
     }
```

Comment:
for my own learning any reason why on error we set the progress bar to invisible but on success we set it to gone?